CIP-xxxx/README.md

```diff
@@ -0,0 +1,145 @@
+---
+CIP: xxxx
+Title: CBOR Tag definition for CIP-0003 ED25519-BIP32 Keys
+Category: MetaData
+Status: Proposed
+Authors:
+    - Steven Johnson<steven.johnson@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/753
+Created: 2024-1-19
+License: CC-BY-4.0
+---
+
+<!-- cspell: words secp -->
+
+<!-- markdownlint-disable MD025-->
+# [CBOR] Tag definition for [CIP-0003] [ED25519-BIP32] Keys
```

Comment:
@rphair correct me if I'm wrong, I believe we do not generally include an H1 tag as this is parsed from the yaml header in downstream renderers.

Reply:
@Crypto2099 you are right; this is something we tell everyone who uses an H1 tag & it's in CIP-0001 here (see **Note**): https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#structure